### DISCUSSION TITLE
HM-MOD-RPI-PCB mit RaspberryMatic

### DISCUSSION THREAD

**Discussion by roland1205** [OpenCCU/OpenCCU#1216] [🇩🇪 Fragen&Antworten]
Hallo,
das Modul mit den Lötpunkten gelötet und aufgesetzt. Leider kann ich keine Geräte einbinden - bin mir nicht sicher, ob überhaupt das Funkmodul gefunden wird.
Hardware ist ein Raspberry PI 4b. Wo kann ich prüfen, ob das Modul gefunden wir, fehlt ein Treiber ggfs.?
Bitte um kurze Hilfe.
Danke

**Comment by jens-maus** [comment]
Bitte mal ein Bild des zusammengelöteten Funkmodul hier präsentieren.
